src/main/java/bio/terra/service/auth/oauth2/GoogleCredentialsService.java

```diff
@@ -0,0 +1,56 @@
+package bio.terra.service.auth.oauth2;
+
+import bio.terra.service.resourcemanagement.exception.GoogleResourceException;
+import com.google.auth.oauth2.GoogleCredentials;
+import java.io.IOException;
+import java.util.List;
+import org.springframework.stereotype.Component;
+
+@Component
+public class GoogleCredentialsService {
```

Comment:
is there a concern that this could get used by different services and you could end up getting state bleed?  That's the problem with doing this in a service and not a static is that you have to account for that now

Reply:
I wonder if we could protect against that by not storing scopes separately but just passing the into `getAccessToken`